src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,12 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball [] balls = new Ball[3];
```

Comment:
better create constant for 3 number, your can read more about constants here - https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names